src/main/scala/uk/gov/nationalarchives/omega/api/repository/AbstractRepository.scala

```diff
@@ -21,17 +21,22 @@
 
 package uk.gov.nationalarchives.omega.api.repository
 
+import org.apache.jena.ext.xerces.util.URI
 import uk.gov.nationalarchives.omega.api.messages.reply.LegalStatus
-import uk.gov.nationalarchives.omega.api.repository.model.AgentEntity
+import uk.gov.nationalarchives.omega.api.messages.request.ListAgentSummary
+import uk.gov.nationalarchives.omega.api.repository.model.{ AgentDescriptionEntity, AgentSummaryEntity }
 
 import scala.util.Try
 
 trait AbstractRepository {
```

Comment:
I am a bit concerned by the lack of Scaladoc. I am wondering what an "AbstractRepository" or even a "Repository" is?

Reply:
Have added Scaladoc here.